apps/prairielearn/src/schemas/schemas/infoAssessment.json

```diff
@@ -194,7 +194,7 @@
         },
         "mode": {
           "description": "The server mode required for access.",
-          "enum": ["Public", "Exam", "SEB"]
```

Comment:
Can this break existing assessments?

Reply:
See the note in the PR description. To be really specific: there are two access rules from the test course and three from the deleted 2018 course instance.